lib/src/date-input/Calendar.tsx

```diff
@@ -39,22 +39,29 @@ const getDateToFocus = (selectedDate, innerDate, today) => {
   return selectedDate?.get("month") === innerDate.get("month") && selectedDate?.get("year") === innerDate.get("year")
     ? selectedDate
     : today.get("month") === innerDate.get("month") && today.get("year") === innerDate.get("year")
-    ? today
-    : innerDate.set("date", 1);
+      ? today
+      : innerDate.set("date", 1);
 };
 
 const isDaySelected = (date: { day: number; month: number; year: number }, selectedDate) =>
   selectedDate?.get("month") === date.month &&
   selectedDate?.get("year") === date.year &&
   selectedDate?.get("date") === date.day;
 
+const chunk = (data: any[], chunkSize: number) => {
```

Comment:
The names feel generic and makes this function difficult to understand.

Reply:
Yup, you are totally right, I might try something like `divideDaysIntoWeeks`